lib/pusher/push_notifications/client.rb

```diff
@@ -30,7 +28,24 @@ def post(resource, payload = {})
             body = JSON.parse(response.body)
             Response.new(status, body, status == 200)
           else
-            Response.new(500, nil, false)
+            Response.new(status, nil, false)
+          end
+        end
+      end
+
+      def delete(resource)
+        url = build_users_url(resource)
```

Comment:
We need to URL encode the user id as we support any UTF-8 characters (e.g. we allow the user id `/`).

Please add a test for this as well.

Reply:
Fixed in 259e79c.